system/autoware_diagnostic_graph_aggregator/src/common/graph/levels.cpp

```diff
@@ -47,7 +55,7 @@ void LatchLevel::update(const rclcpp::Time & stamp, DiagnosticLevel level)
 {
   input_level_ = level;
 
-  if (latch_enabled_) {
+  if (latch_enabled_ && !initializing_) {
```

Comment:
Ah, I understand why the initialization flag is needed. We need to update the status only when initialization is done.